src/components/Api.js

```diff
@@ -0,0 +1,118 @@
+export default class Api {
+  constructor({ baseUrl, headers }) {
+    // constructor body
+    this._baseUrl = baseUrl;
+    this._authorization = headers.authorization;
+    this._contentType = headers["Content-Type"];
+  }
+
+  _checkResponse(res) {
+    if (res.ok) {
+      return res.json();
+    }
+  }
+  getInitialCards() {
+    return fetch(`${this._baseUrl}/cards`, {
+      headers: {
```

Comment:
Needs correction: you should store the headers object in the constructor and reuse it in all the api methods:

```js
headers: this._headers
```